app/views/shared/_bootstrap_tree.html.haml

```diff
@@ -0,0 +1,7 @@
+#bootstrap-tree{:style => "padding-left: 17px; padding-right: 30px", "ng-controller" => "bootstrapTreeController"}
```

Comment:
This is only for the left side nav, so yeah maybe `#bootstrap-tree-left-nav` or something would be better.